net/snort3/files/snort.init

```diff
@@ -38,17 +38,30 @@ start_service() {
 	if [ "$manual" = 0 ]; then
 		local config_file=$($MGR setup)
 		procd_set_param command "$PROG" -q -c "${config_file}"
+		jail
 	else
 		procd_set_param command $PROG -q -i "$interface" -c "${config_dir%/}/snort.lua" --tweaks local
 		procd_set_param env SNORT_LUA_PATH="$config_dir"
 		procd_set_param file $CONFIGFILE
+		jail
 	fi
 	procd_set_param respawn
 	procd_set_param stdout 0
 	procd_set_param stderr 1
 	procd_close_instance
 }
 
+jail()
+{
+	[ -x /sbin/ujail -a -e /etc/capabilities/snort.json ] && {
```

Comment:
Would this check ever be false? Can `ujail` no be present? Aside from the second condition and from a user deleting the caps file.

Reply:
I believe the answer to this question is yes. If I user either did not have `procd-ujail` installed or if `/sbin/ujail` was inaccessible due to user error (for example deleting it for some reason).